ee/clickhouse/views/test/test_clickhouse_insights.py

```diff
@@ -78,9 +109,86 @@ def test_funnel_trends_basic_post(self):
                 "date_from": "2021-05-01 00:00:00",
                 "date_to": "2021-05-07 23:59:59",
                 "funnel_viz_type": "trends",
+                "funnel_order_type": "ordered",
```

Comment:
oh, this will break existing funnel-trends API, since right now no-one sends `funnel_order_type`, and without it, the default class is `ClickhouseFunnel`, not `ClickhouseFunnelNew`, which means bad things happen.

Reply:
Not true anymore, since there's no `FunnelNew` anymore.